README.md

```diff
@@ -99,22 +99,26 @@ The Prefab5 connection repository doesn't have a default value. The Symfony DI s
 
 ### Rescheduling Decorator
 
-This decorator is intended for jobs rescheduling themselves (observers). It completes the running job with success and schedules the next, possibly delayed. The completion and scheduling is atomic, since it's done in a transaction, which is why the decorator needs to know what connection is used by Kōjō.  
-The decorator doesn't reschedule if the decorated worker has already applied a state change request, e.g. has held or has completed with success. If that is the case, the decorator logs a warning, which is why it's recommended to have this decorator at the top of the decorator stack (in the worker builder's service definition/YAML file).
+This decorator is intended for jobs rescheduling themselves (observers). It completes the running job with success and schedules the next, possibly delayed.
+
+The completion and scheduling is atomic, since it's done in a transaction, which is why this decorator needs to know what connection is used by Kōjō. Use the [Userland PDO Decorator](#userland-pdo-decorator) to set the Kōjō connection. The values of the connection related DI parameters for the Rescheduling Decorator should match the values of the corresponding DI parameters for the Userland PDO Decorator.
+
+The decorated worker shouldn't complete with success since the Rescheduling Decorator does.  
+The decorator doesn't reschedule if the decorated worker has already applied a state change request, e.g. has held or has completed with success. If that is the case, the decorator logs a warning.  
+The warning will be logged if another decorator between the Rescheduling Decorator and Worker applies a state change request, for example the [Retry Threshold Decorator](#retry-threshold-decorator) due to too many retries. Therefore, it's recommended to have the Rescheduling Decorator as innermost, i.e. at the top of the decorator stack in the worker builder's service definition/YAML file, like it's done in the [example](https://github.com/neighborhoods/KojoWorkerDecoratorComponentFitness/blob/main/src/ReschedulingDecorator/Worker/Builder.service.yml#L9).
 
 #### Parameters
 
 * **jobTypeCode**  
   Type: string  
-  Job type code of the scheduled job. Doesn't have a default value and has to be provided.
+  Job type code of the scheduled job. Doesn't have a default value. This should be the worker's job type code if the job reschedules itself.
```

Comment:
Relatedly: would it be useful for the kojo worker service API to expose the job type? It would come in useful for a bunch of these decorators, but I don't know if you want to go through the trouble of supporting kojo versions that expose it and those that don't, and having branching logic to get that value

Reply:
It could also just have the KWDC 1.1 require Kojo 5.1. I don't think I'd consider that a breaking change.

Reply:
> I'm curious: do you foresee uses of this decorator that would reschedule a job of a different type?

The intention is rescheduling. But since the user specifies the job type code manually, they could really put anything in there. I cannot think of a concrete use case where the job type code would differ from the current job, mainly because I cannot imagine a job using this decorator if it isn't an observer.
If there were use cases where the job type code can differ, this decorator would not _reschedule_, but simply _schedule_. So it's name is inappropriate.

Would you like me to made a change to the readme before merging?